dev/com.ibm.ws.jaxrs.2.1.cdi.2.0_fat/test-applications/atinjectapp/src/jaxrs21/fat/atinject/AtInjectDirectService.java

```diff
@@ -26,7 +26,7 @@
     private FieldInjectedObject field;
     private AbstractInjectedObject method;
 
-    //TODO: determine why no injection occurs unless this default ctor exists
+    //The CDI spec indicates that a no-arg ctor is required
```

Comment:
Can you add the specific sections from the spec - I think that will help us when users ask about this in the future.